src/util/circularbuffer.h

```diff
@@ -14,16 +15,14 @@ class CircularBuffer {
   public:
     CircularBuffer(unsigned int iLength)
             : m_iLength(iLength),
-              m_pBuffer(new T[m_iLength]),
+              m_pBuffer(std::vector<T>(m_iLength)),
```

Comment:
m_pBuffer{m_iLength}

Reply:
[The "Uniform Initialization Syntax" is not entirely "uniform"...](https://github.com/mixxxdj/mixxx/pull/4341/files#r720106286)